### ISSUE TITLE
Add field in user profiles to support BlueSky accounts

### ISSUE THREAD

**Issue by froggleston** [carpentries/amy#2726]
Currently AMY only supports Twitter/X social media accounts. We should add BlueSky as this is becoming much more popular with scientists and researchers.

Relates to https://github.com/carpentries/carpentries.org/issues/310

**Comment by maneesha** [comment]
Re-opening this as the field has been added to the "edit" view but not to the "view" view.  
@pbanaszkiewicz  is it [this template](https://github.com/carpentries/amy/blob/develop/amy/templates/workshops/person.html) that it needs to be added to?

**Comment by pbanaszkiewicz** [comment]
@maneesha Yes, that's the template. I'm going to release v4.6.4 over the weekend (or sooner), I can fix this issue then.
